### PR TITLE
 https://issues.redhat.com/browse/ACM-4963 Update search_intro.adoc

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-4963

### CODE REVIEW

File: observability/search_intro.adoc [hunk 1]
```diff
@@ -105,24 +105,6 @@ PostgreSQL supports database tuning to optimize your database performance. The t
...
-----
-apiVersion: search.open-cluster-management.io/v1alpha1
-kind: Search
-metadata:
-  name: search-v2-operator
-  namespace: open-cluster-management
-  labels:
-    cluster.open-cluster-management.io/backup: ""
-spec:
-  dbConfig: tuning-config
```
Comment (SherinV) [inline observability/search_intro.adoc:120]: There is one more example on this page with this line <dbConfig: tuning-config>. Please remove that too.

File: observability/search_intro.adoc [hunk 2]
```diff
@@ -105,24 +105,6 @@ PostgreSQL supports database tuning to optimize your database performance. The t
 oc create configmap tuning-config --from-literal POSTGRESQL_SHARED_BUFFERS=128MB --from-literal POSTGRESQL_EFFECTIVE_CACHE_SIZE=128MB --from-literal WORK_MEM=64MB
 ----
 +
```
Comment (SherinV) [inline observability/search_intro.adoc:100]: Remove the heading - PostgreSQL database configuration: and the remaining lines below that.

File: observability/search_intro.adoc (hunk 1 above)
Comment (dockerymick) [reply observability/search_intro.adoc:120]: So I should also remove 108-157

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

Looks like a linter issue is there. Might just be in your branch and might resolve itself. I don't think it is in your file.

File: observability/search_intro.adoc [hunk 3]
```diff
@@ -96,84 +96,7 @@ spec:
...
-  labels:
-    cluster.open-cluster-management.io/backup: ""
-spec:
-  dbConfig: tuning-config
-  dbStorage:
-  size: 10Gi
-  storageClassName: gp2-search
-----
-+
-- Optimize cost by tuning the pod memory or CPU requirements, replica count, and update log levels for any of the four search pods (indexer, database, queryapi, or collector pod). Update the `deployment` section of the `search-v2-operator` custom resource. There are four deployments managed by the `search-v2-operator`, which can be updated individually. Your `search-v2-operator` custom resource might resemble the following file:
```
Comment (SherinV) [inline observability/search_intro.adoc:126]: Don't remove these lines 126-176. Just remove line 136. `dbConfig: tuning-config`

**Review by SherinV — APPROVED** [review]

**Pull request merged**